### ISSUE TITLE
ambiguous use of '>' for property pages

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#2822]
Considering https://schema.org/itemListElement and similar,

@RichardWallis could you tweak them to be

```
 Thing > Property :: itemListElement 
```

instead of 

```
 Thing > Property > itemListElement 
```

... since itemListElement is of type Property, rather than a subclass of it.

"::" isn't ideal, but we use it on the Enumeration pages in a comparable manner.

**Comment by danbri** [comment]
...oooOOO('Thing ɟOssɐʃↃqns Property ǝdʎʇ identifier ɟOʎʇɹǝdoɹԀqns isbn')

**Comment by RichardWallis** [comment]
Updated thus:
```Thing > Property :: identifier```
and
```Thing > Property :: identifier : isbn```

in commit: dea1377

**Comment by danbri** [comment]
thanks!

**Comment by RichardWallis** [comment]
Implemented in V12.0
